src/protocol/abi/json_abi_format.md

```diff
@@ -49,6 +49,11 @@ The ABI of a contract is represented as a JSON object containing the following p
     - `"typeArguments"`: an array of the _type arguments_ used when applying the type of the output, if the type is generic, and `null` otherwise. Each _type argument_ is a _type application_ represented as a JSON object that contains the following properties:
       - `"type"`: the _type declaration_ ID of the type of the _type argument_.
       - `"typeArguments"`: an array of the _type arguments_ used when applying the type of the _type argument_, if the type is generic, and `null` otherwise. The format of the elements of this array recursively follows the rules described in this section.
+  - `"attributes"`: an optional array of _attributes_. Each _attribute_ is represented as a JSON object that contains
+  the following properties:
+    - `"name"`: the name of the attribute (the only valid attribute names are `storage`, `payable`, `test`, `inline`,
```

Comment:
I propose that we make this sentence less specific about what attributes are supported, and move the actual supported attributes into a separate section in this file where each attribute is listed with exactly what it means and what it implies (e.g. `payable` implies that no coins can be forwarded when calling this method, etc.). We can point to that section here by saying: "For a list of supported attributes, see ..."

Reply:
Great suggestion -- I took a stab at it